user/languages/php.md

```diff
@@ -200,30 +200,6 @@ To see real world examples, see:
 - [LiipHyphenatorBundle](https://github.com/liip/LiipHyphenatorBundle/blob/master/.travis.yml)
 - [doctrine2](https://github.com/doctrine/doctrine2/blob/master/.travis.yml)
 
-### Installing PEAR packages
-
-If your dependencies include PEAR packages, the Travis CI PHP environment has the [Pyrus](http://pear2.php.net/) and [pear](http://pear.php.net/) commands available:
```

Comment:
are the `pear` commands not available either? (I don't know)

Reply:
`pear` _might_ be, but I suppose those who are still using it will know the risks of being deprecated.